hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheService.java

```diff
@@ -435,7 +435,8 @@ public CacheStatisticsImpl getStatistics(String name) {
     @Override
     public CacheOperationProvider getCacheOperationProvider(String nameWithPrefix, InMemoryFormat inMemoryFormat) {
         if (InMemoryFormat.NATIVE.equals(inMemoryFormat)) {
-            throw new IllegalArgumentException("Native memory is available only in Enterprise!");
+            throw new IllegalArgumentException("Native memory is available only in Hazelcast Enterprise." +
```

Comment:
I think the `+` has to go into a new line, if CheckStyle is active for this class.